Motorcycle.js

```diff
@@ -1,6 +1,26 @@
 var Car = require('./Car.js');
 
-//Implement your Motorcycle here
-function Motorcycle() {};
+// call motorcycle from car
+function Motorcycle(make, model, year, color, seats, passengers){
+	Car.call(this, make, model, year, color, seats, passengers);
+	if (this.seats > 2) {
+		console.log("Motorcycles cannot have more than two seats!")
+		return false
```

Comment:
Here, instead of ```return false```, we'll need to override the number of seats specified by the user. We can do that by simply declaring the following:
```
this.seats = 2;
```